index.js

```diff
@@ -26,6 +26,19 @@ if (!rules) {
   );
 }
 
+function getConfig(type) {
+  return {
+    plugins: ["local-rules"],
+    rules: Object.fromEntries(rules.map((rule) => {
```

Comment:
This doesn't work. `rules` is an object of the form:

```
{
  'disallow-identifiers': {
    // ...
  },
  'another-rule': {
    // ...
  },
}
```

which doesn't have a `map` method. `Object.keys(rules).map` should work.